<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorAreaMaintain</class>
 <widget class="QWidget" name="MonitorAreaMaintain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1061</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(235, 235, 235, 255), stop:1 rgba(170, 170, 170, 255));</string>
   </property>
   <widget class="QLabel" name="titlePicLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>监控区域信息维护</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="contentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>1061</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(200,200,200);
border-bottom-left-radius:5px;
border-bottom-right-radius:5px;
background-color:rgb(236,236,236);</string>
   </property>
   <widget class="QLabel" name="numberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>区域编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>区域名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>区域类型</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeLlineEdit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="areaLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>行政区划</string>
    </property>
   </widget>
   <widget class="QComboBox" name="areaComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none; 
border-radius:0px;</string>
    </property>
    <property name="text">
     <string>区域状态</string>
    </property>
   </widget>
   <widget class="QComboBox" name="statusComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:0px;
background-color:rgb(255,255,255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border：1px solider rgb(255,255,255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>+添加区域</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1021</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="allSelPushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="allClePushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>反选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
